Makefile

```diff
@@ -0,0 +1,77 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+tag=incubator-zeppelin
+littleName=iz
+currentDir=$(shell pwd)
+
+install = docker run -ti --rm -v "$(HOME)/.m2:/root/.m2" -v "$(currentDir):/root/incubator-zeppelin" -u root -w /root/incubator-zeppelin --name $(littleName) $(tag) mvn clean install -DskipTests $(1)
+
+build:
+	docker build -t $(tag) .
+rm:
+	docker rm -f $(littleName)
+stop:
+	docker stop $(littleName)
+rmi:
+	docker rmi -f $(tag)
+logs:
+	docker logs $(littleName)
+run:
+	docker run -ti --rm -p 8080:8080 -v "$(HOME)/.m2:/root/.m2" -v "$(currentDir):/root/incubator-zeppelin" -u root -w /root/incubator-zeppelin --name $(littleName) $(tag) /bin/bash
+
+install-local:
+	$(install)
+
+install-Spark-1.5.x:
```

Comment:
Is this scalable solution? What if profiles are removed/added? Are those goals really needed? readme explains building via profiles, while this makefiles takes different approach, maybe it make sense to use only single approach?